### ISSUE TITLE
How to test the pictures provided by myself

### ISSUE THREAD

**Issue by zw-92** [Shantanu48114860/MTL-TCNN3#1]
Hello, your idea is great. I want to use it for my own test pictures. What should I do? We look forward to your reply.

**Comment by shantanu-ai** [comment]
So u have a dataset right? Your own dataset?

**Comment by zw-92** [comment]
Yes, I want to use the model directly for testing, not training.

**Comment by shantanu-ai** [comment]
Ok, so u should walk through from main_texture file mentioned in the readme. Then go to the file it redirects, there u will find a bunch of methods like one for dtd, kth datasets etc. so create your own method, then probably u need to provide the names of ur own labels for textures and the labels for imagenet will be same as mentioned for other datasets and please run then.

Its been a long time since i developed this code.

**Comment by shantanu-ai** [comment]
I developed each module from scratch, one thing i would suggest is to use a good backbone like resnet 50 for feature extraction for the common part, u will see better results.
